src/proxy/wrap_server_transport.rs

```diff
@@ -3,10 +3,10 @@ use tokio::io::{AsyncRead, AsyncWrite};
 use super::Source;
 
 /// Wraps serverside transports with additional functionality.
```

Comment:
I didn't find the purpose of this very obvious right away. "Wrapping with additional functionality" seems like it could helpful, but if a case doesn't come to mind right away it seems a little vague?

Could this comment provide a quick explanation on what an `io` actually does with `Source`?

Reply:
Yeah, I'm removing this in the next PR.

Reply:
Sounds good